lib/dfe/analytics/services/postgres_checksum_calculator.rb

```diff
@@ -65,7 +67,8 @@ def build_select_and_order_clause(order_column, table_name_sanitized)
         def build_where_clause(order_column, table_name_sanitized, checksum_calculated_at_sanitized)
           return '' unless WHERE_CLAUSE_ORDER_COLUMNS.map(&:downcase).include?(order_column.downcase)
 
-          "WHERE DATE_TRUNC('milliseconds', #{table_name_sanitized}.#{order_column.downcase}) < DATE_TRUNC('milliseconds', #{checksum_calculated_at_sanitized}::timestamp)"
+          # Add IS NULL to include records with null updated_at / created_at values
+          "WHERE (#{table_name_sanitized}.#{order_column.downcase} IS NULL OR DATE_TRUNC('milliseconds', #{table_name_sanitized}.#{order_column.downcase}) < DATE_TRUNC('milliseconds', #{checksum_calculated_at_sanitized}::timestamp))"
```

Comment:
@ericaporter will this addition ever actually do anything, since if it is true then the WHERE clause wouldn't operate, or would be switched to created_at if created_at does not contain null values?

Reply:
I think that's fair, this was the first thing I updated but it won't do anything if we are excluding order columns with null values, as much as I like a belt and braces approach...

Reply:
OK to keep as belt and braces I believe.